tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -455,6 +458,14 @@ setup_selinux() {
 
 setup_systemd() {
 		local mount_dir="$1"
+		if [ "${DM_VERITY}" == "no" ]; then
```

Comment:
nit: I know that `set -o nounset` is disabled on this script but If one days it gets enabled then that line will fail. So could you initialize it on begin of the script just like the other variables (e.g. MEASURED_ROOTFS).

Reply:
Fixed. Thanks.